src/compiler/worker/initWorker.js

```diff
@@ -82,7 +89,7 @@ module.exports = function initWorker({
       if (fs.existsSync(filename)) {
         send(Events.FILE_RECEIVED, {
           taskId: payload.taskId,
-          file: fs.readFileSync(filename),
+          file: filename,
```

Comment:
I think it makes sense now to rename the field to `filename` or `filePath`, pick whatever feels better :)